src/Application/SLADEWxApp.cpp

```diff
@@ -771,7 +771,7 @@ CONSOLE_COMMAND(crash, 0, false)
 		== wxYES)
 	{
 		uint8_t* test = nullptr;
-		test[123]     = 5;
```

Comment:
Going a bit further, this could be improved by making it volatile, since the compiler may decide that this is nonsense and to ignore it.

`abort()` may even be better, but that results in a different reror.